charts/amazon-cloudwatch-observability/templates/operator-clusterrole.yaml

```diff
@@ -5,19 +5,31 @@ metadata:
 rules:
 - apiGroups: [ "" ]
   resources: [ "configmaps" ]
-  verbs: [ "create", "delete", "get", "list", "patch", "update", "watch" ]
+  verbs: [ "create","get","list", "watch" ]
+- apiGroups: [ "" ]
+  resources: [ "configmaps" ]
+  resourceNames: ["cloudwatch-agent", "cloudwatch-agent-windows", "cwagent-clusterleader", "dcgm-exporter-config-map", "fluent-bit-config", "fluent-bit-windows-config", "neuron-monitor-config-map", "kube-root-ca.crt"]
```

Comment:
Restricting these by names is going to be tricky.
Customers installing their own AmazonCloudWatchAgent CR can set the `name` to be anything and the resources would get named accordingly per the [regexes](https://github.com/aws/amazon-cloudwatch-agent-operator/blob/099460aea6622b73557017a14b5c46e1b10de680/internal/naming/main.go).

Reply:
Also, curious, why do we need the `kube-root-ca.crt` in this list?

Reply:
yeah, I was hoping there would be a way to do this by namespace, but it doesn't seem like thats possible. What if we pull the actual names from the values.yaml?

`kube-root-ca.crt` was found in the `amazon-cloudwatch` namespace, so I assume the operator should have domain over anything we create via helm/operator

Reply:
>What if we pull the actual names from the values.yaml?

In the scenario of customers installing their own CRs, it wouldnt come from the values. But perhaps thats just setting an expectation that they have to modify the RBAC accordingly when they do such customizations.

Reply:
>kube-root-ca.crt was found in the amazon-cloudwatch namespace, so I assume the operator should have domain over anything we create via helm/operator

Interesting. Is it actually created/modified by our service account though?